### ISSUE TITLE
coordination of pull requests

### ISSUE THREAD

**Issue by championswimmer** [coding-blocks/dev-submit#31]
@abhishek1208 @tech4GT @apoorvaagupta @bhavyaagg @anu0012

Some issues are created in such a way, that they need to be solved in a particular order, or else you'll have merge conflicts.

For eg, you have to decide if you want to refactor db first, and then refactor the routes, or the other way round.
Since more than 1 person is involved, what you need to do is, first comment on the issue, and mention that fact like this - 
```
I am going to work on issue #X first. Since issue #Y depends on #X, please do not solve #Y until #X is resolved
```

Whenever a new issue is created, and you can see that there is a sequential dependency on another issue, this discussion must be done first. 
There is a reason Github allows commenting on issues. Just assigning someone to an issue is not the right way either. It is important to mention in comments that which order you have planned to refactor. 

I want a comment from each of the tagged person here underneath, and only then will I close this issue. And before this issue is closed, no other PR is going to be merged into this repository.

**Comment by abhishek1208** [comment]
issue #20 must be resolved first since the changes done in the model would create conflicts
issue #21
issue #19 
issue #12 
issue #13 
This must be the order of the issues since it involves a major change in the model structure!

**Comment by apoorvaagupta** [comment]
Yes sir! I would keep this in my mind in future.

**Comment by bhavyaagg** [comment]
Yes sir! Won't happen again.

**Comment by anu0012** [comment]
Sure sir! I'll take care of this in future.
